### DISCUSSION TITLE
How to configure multiple LLM nodes in one flow?

### DISCUSSION THREAD

**Discussion by buyangnie** [FlowiseAI/Flowise#2437] [General]
In complicated workflow,  I need to configure more than one LLM node.  
For example, query-> LLM->output->LLM->output, 
but it looks like only one runable node (such as LLMChain, Retrieval-QA Chain, etc, ) can run.
How to link several Chain objects with each others in workflow?

**Comment by timur-nocodia** [comment]
You can set Output of LLM Chain node to Output Prediction and connect it to a prompt node to the Format Prompt Values
then you can create a Format Prompt Values and choose the source for that variable.

![image](https://github.com/FlowiseAI/Flowise/assets/120160081/e97475a6-55bd-45dc-99e2-cd19dab30d3d)

and feed it to another LLM as a prompt and thats the chain.
